CodePushNativePlatformAdapter.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+var NativeCodePush = require("react-native").NativeModules.CodePush;
+
+var Platform = require("Platform");
+var Alert;
+
+if (Platform.OS === "android") {
+  /* 
+   * Promisify native methods. Assumes that every native method takes
+   * two callback functions, resolve and reject.
+   */
+  var methodsToPromisify = [
+    "installUpdate",
+    "downloadUpdate",
+    "getConfiguration",
+    "getCurrentPackage",
+    "isFailedUpdate",
+    "isFirstRun",
+    "notifyApplicationReady",
+    "setDeploymentKey"
+  ];
+  
+  methodsToPromisify.forEach((methodName) => {
```

Comment:
Do we need to do this now that https://github.com/facebook/react-native/commit/b86a6e3b44a63e92cf3a7976d2fa26c4bf412df1#commitcomment-14069903 went in to master?